chart/templates/ingress_route.yaml

```diff
@@ -1,17 +1,25 @@
-{{- if .Values.web.ingress.enabled }}
+{{- if .Values.web.external_service.enabled }}
 ---
+{{- $root := . -}}
+{{- range $name, $service := $root.Values.web.external_service.service }}
 apiVersion: traefik.containo.us/v1alpha1
 kind: IngressRoute
 metadata:
-  name: {{ .Values.appName }}
-  namespace: {{ .Release.Namespace }}
+  name: {{ $name }}
+  namespace: {{ $root.Release.Namespace }}
 spec:
   entryPoints:
     - web
   routes:
-    {{- range .Values.web.ingress.routes }} # Iterating over a Complex Structure
-    - {{ . | toYaml | nindent 6 -}}
-    {{- end }}
+    - kind: {{ $root.Values.web.external_service.default.kind }}
```

Comment:
I seems that we would want to iterate over the various ingress routes here but maybe i'm wrong

Reply:
AH I see on line 4 sorry!

Reply:
Yep! Consider `vets-api-web` as a service to make it easier to loop and apply conditional that do not apply to `stats`